docs/source/getting-started.rst

```diff
@@ -264,12 +264,11 @@ duration.
     private final Timer responses = metrics.timer(name(RequestHandler.class, "responses"));
 
     public String handleRequest(Request request, Response response) {
-        final Timer.Context context = responses.time();
-        try {
+        try(final Timer.Context context = responses.time()) {
             // etc;
             return "OK";
         } finally {
-            context.stop();
+            // doSomething();
```

Comment:
nit: Can we remove the finally block altogether?

Reply:
@dennyac I have replaced finally with comment. Is that fine?

Reply:
Not super opinionated on it. Will let the maintainers decide.